pkg/controller/clustersync/clustersync_controller_test.go

```diff
@@ -576,15 +394,10 @@ func TestReconcileClusterSync_ApplySecret(t *testing.T) {
 			srcSecret := testsecret.FullBuilder(testNamespace, "test-secret", scheme).Build(
 				testsecret.WithDataKeyValue("test-key", []byte("test-data")),
 			)
-			rt := newReconcileTest(t, mockCtrl, scheme,
-				cdBuilder(scheme).Build(),
-				clusterSyncBuilder(scheme).Build(),
-				teststatefulset.FullBuilder("hive", stsName, scheme).Build(
-					teststatefulset.WithCurrentReplicas(3),
-					teststatefulset.WithReplicas(3),
-				),
-				syncSet,
-				srcSecret)
+			rt := func() *reconcileTest {
```

Comment:
ntr: Well, I was just trying to de-lint this thing to remove unused vars (t, scheme). I used a vscode widget to do the refactor... and it did this (throughout). It's not _wrong_, but it's kinda goofy :(

Reply:
Looking at it again today, I'm actually liking the refactored version. Certainly not worth the churn to convert it back. Going to leave it alone.